constants/cancun/constants.zkasm

```diff
@@ -1 +1,3 @@
-const MAX_PRC_ADDRESS = 0x0a
\ No newline at end of file
+const MAX_PRC_CONTIGUOUS_ADDRESS = 0x0a
+;; this constant will be used once issue in go-corset is fixed https://github.com/Consensys/go-corset/issues/1284
+const P256_VERIFY_ADDRESS = 0x0a ;; P256_VERIFY does not exist in Cancun, we set to an already checked precompile address to avoid duplicating trm module
```

Comment:
I am tempted to say not to bother putting in the `P256_VERIFY_ADDRESS` until the issue is fixed.  Just leave a note (as you have) in `trm_london.zkasm` or something.  Otherwise, its a bit confusing to have constants which are not used.  It could be a while before we have time to fix the issue, etc.